Source/Eto.Gtk/Forms/Controls/SplitterHandler.cs

```diff
@@ -114,6 +114,38 @@ void Create()
 			}
 			if (position != null)
 				Control.Position = position.Value;
+
+			if (fixedPanel != SplitterFixedPanel.Panel1)
+				Control.SizeAllocated += Control_SizeAllocated;
```

Comment:
This may be called more than once, causing the event to be subscribed multiple times